packages/repack/js/NativeScriptManager.ts

```diff
@@ -0,0 +1,47 @@
+import type {TurboModule} from 'react-native';
+import {TurboModuleRegistry} from 'react-native';
+import type { Int32 } from 'react-native/Libraries/Types/CodegenTypes';
+
+type WebpackContext = {
+    p: () => string;
+    u: (id: string) => string;
+}
+
+type NormalizedScriptLocator = {
+    method: string;
+    url: string;
+    fetch: boolean; 
+    timeout: Int32;
+    absolute: boolean;
+    query?: string;
+    headers?: string;
+    body?: string;
+    token?: string;
```

Comment:
`token` was removed entirely in recent update to CodeSigning: https://github.com/callstack/repack/pull/348/files#diff-6e569776db8e37a5482958fa2218c1d9fa5353c38c9de03df173a5b830bbce36L20-L21